DED-520 Updating supervisor app readme v2 (#27)
livechat/supervisor-app: README.md

2. Click **New App** and give it an **App Name**.
3. Choose the **LiveChat** product as the product you want to build for.
4. Go to **Building Blocks**.
5. Add **App Authorization** and mark it as *JavaScript App*. Your **Client Id** will be displayed there.
6. Fetch the **Supervisor** app repository.
7. In the app directory, do the following steps :

    * Install dependencies (`npm install`).
    * In your project, go to `src/utils/congif.js` and replace `client_id` with your own **Client Id** (the one from **Step 5**).
    * Run your app (`npm start`).
8. Add your app url (for example: `https://localhost:3000`) in these two locations:

    * **Redirect URI whitelist**
    * **Agent App Widgets**
9. In **Private installation**, click **Install app**. 

You should now be able to use **Supervisor** with LiveChat.

# How it works

[Agent App Widgets](https://developers.livechat.com/docs/extending-agent-app) are web applications loaded inside the LiveChat Agent App. All agents can interact with the widget during chats with customers. The widget itself is displayed in the Agent’s App sidebar.

> To get information such as tags and canned responses, you need to use [LiveChat Rest API](https://developers.livechat.com/docs/management/configuration-api/v2.0/).

Author: As far as I checked, canned responses are still only in Config API v2 so I believe this should be still linked here

Author: Okay, let's do it this way. I left it like that because of the canned responses, but if we can drop them, then I will link new Conf API

> To get information such as tags and canned responses, you need to use [LiveChat Rest API](https://developers.livechat.com/docs/management/configuration-api/v2.0/).

Author: ```suggestion
To get information such as tags, you need to use [Configuration API](https://developers.livechat.com/docs/management/configuration-api).
```

> In order to pull data from our server, Rest API requires you to include an **access_token** in all the requests. You can get it using [Personal Access Tokens](https://developers.livechat.com/docs/authorization/agent-authorization#personal-access-tokens) and [JavaScript Widget API](https://docs.livechatinc.com/agent-app-widgets/#javascript-api).

Author: changed [LiveChat Boilerplate](https://docs.livechatinc.com/boilerplate/) for [Personal Access Tokens](https://developers.livechat.com/docs/authorization/agent-authorization#personal-access-tokens) as I believe that is the correct one now.

> In order to pull data from our server, Rest API requires you to include an **access_token** in all the requests. You can get it using [Personal Access Tokens](https://developers.livechat.com/docs/authorization/agent-authorization#personal-access-tokens) and [JavaScript Widget API](https://docs.livechatinc.com/agent-app-widgets/#javascript-api).

Reviewer: ```suggestion
In order to pull data from our server, Rest API requires you to include an **access_token** in all the requests. You can get it using one of the [agent authorization flows](https://developers.livechat.com/docs/authorization/agent-authorization).```

> You should also get familiar with [Authorization](https://docs.livechatinc.com/authorization/).

Reviewer: ```suggestion
```

# More sample apps 
Experiment more with two different sample apps:
 - [Tag Master](https://github.com/livechat/tag-master) - widget for helping your agents manage tags and canned responses.
 - [Progress](https://github.com/livechat/progress-app) - widget for helping you monitor statistics of your team, such as chat ratings, response times, and chatting times.

# Feedback
If you find some bugs, please create an issue in this repo. We will try to fix it ASAP ;)

> # If you're new to LiveChat

Reviewer: How about we link Discord somewhere here?

> **LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 37,000 companies all over the world. For more info, check out [LiveChat for Developers](https://developers.livechatinc.com/).

Author: ```suggestion
**LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 37,000 companies all over the world. Read more about [LiveChat for Developers](https://developers.livechatinc.com/) and [join our Discord](https://discord.com/invite/NcfJu3a9kM) to learn, get inspired, and meet other developers.
```

7. In the app directory, do the following steps :

    * Install dependencies (`npm install`).
    * In your project, go to `src/utils/congif.js` and replace `client_id` with your own **Client Id** (the one from **Step 5**).
    * Run your app (`npm start`).
8. Add your app url (for example: `https://localhost:3000`) in these two locations:

    * **Redirect URI whitelist**
    * **Agent App Widgets**
9. In **Private installation**, click **Install app**. 

You should now be able to use **Supervisor** with LiveChat.

# How it works

[Agent App Widgets](https://developers.livechat.com/docs/extending-agent-app) are web applications loaded inside the LiveChat Agent App. All agents can interact with the widget during chats with customers. The widget itself is displayed in the Agent’s App sidebar.

To get information such as tags, you need to use [Configuration API](https://developers.livechat.com/docs/management/configuration-api).

In order to pull data from our server, you need to include an **access_token** in all the requests. You can get it using one of the [agent authorization flows](https://developers.livechat.com/docs/authorization/agent-authorization).

# More sample apps 

> Experiment more with two different sample apps:

Author: ```suggestion
Experiment more with our different sample apps:
```

 - [Tag Master](https://github.com/livechat/tag-master) - widget for helping your agents manage tags and canned responses.
 - [Progress](https://github.com/livechat/progress-app) - widget for helping you monitor statistics of your team, such as chat ratings, response times, and chatting times.

> - [Sample auth app with the redirect flow](https://github.com/livechat/sample-app-redirect-auth)

Author: ```suggestion
 - [Sample auth app with the redirect flow](https://github.com/livechat/sample-app-redirect-auth)
```

> - [Sample authapp with the popup flow](https://github.com/livechat/sample-app-popup-auth)

Author: ```suggestion
 - [Sample authapp with the popup flow](https://github.com/livechat/sample-app-popup-auth)
```

>  - [Sample authapp with the popup flow](https://github.com/livechat/sample-app-popup-auth)

Author: ```suggestion
 - [Sample auth app with the popup flow](https://github.com/livechat/sample-app-popup-auth)
```
